portfolio/src/main/webapp/style.css

```diff
@@ -92,15 +92,14 @@
   z-index: 0;
 }
 
-.content > p {
+.content * {
```

Comment:
In general, try to avoid using wildcard selector. Here I think `.content` alone will do the same.

Reply:
Fixed. Is there any way I can specifically change the margin of the p element within the .content CSS? Also, I added a color item to the a element because it was not inheriting from the .content style, I'm not sure why.

Reply:
`.content > p` should be able to set the margin of `p` element correctly. If it doesn't look like what you expected, try to use "inspect" function in your browser to check what exactly the style for each element.

"I added a color item to the a element"
do you refer to the `<a>` element? This tag has its own "link" color unless you explicitly overwrite it.

Reply:
Ok, I see. Thank you!